client/src/lib.rs

```diff
@@ -0,0 +1,107 @@
+use anyhow::Context;
```

Comment:
Maybe. I also not sure `utils` is a good fit. Humm I guess it could go there for now, and if we add more to it later we could move it to its own crate.

Reply:
On the other hand, I can see some general utility to a single espresso-client package that someone could install from crates.io and use to interact with sequencer API.